src/api/myComments.ts

```
@@ -0,0 +1,13 @@
+import { authApi } from '@/api/instance/authApi'
+import { MyCommentsResponse } from '@/types/api/myComments'
+
+export const fetchMyComments = async (sort: string = 'latest') => {
+  try {
+    const res = await authApi.get<MyCommentsResponse[]>(
+      `/comments/mine?sort=${sort}`,
+    )
+    return res.data
+  } catch (error) {
+    throw error
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)